README.md

```diff
@@ -33,54 +34,54 @@ By default, the prefix is `/lambda/mipsSecret`, resulting the following required
-| --- | --- | --- |
-| CacheTTL | CacheTTL | TTL value for CloudFront cache objects. |
-| MipsOrganization | MipsOrg | Log in to this organization in the finance system. |
-| SsmParamPrefix | SsmPath | Path prefix for required secure parameters. |
-| CodesToOmit | CodesToOmit | List of numeric codes to treat as inactive. |
-| NoProgramCode | NoProgramCode | Numeric code to use for "No Program" meta-program. |
-| OtherCode | OtherCode | Numeric code to use for "Other" meta-program. |
+| Template Parameter | Environment Variable | Description                                        |
+| ------------------ | -------------------- | -------------------------------------------------- |
+| CacheTTL           | CacheTTL             | TTL value for CloudFront cache objects.            |
+| MipsOrganization   | MipsOrg              | Log in to this organization in the finance system. |
+| SsmParamPrefix     | SsmPath              | Path prefix for required secure parameters.        |
+| CodesToOmit        | CodesToOmit          | List of numeric codes to treat as inactive.        |
+| NoProgramCode      | NoProgramCode        | Numeric code to use for "No Program" meta-program. |
+| OtherCode          | OtherCode            | Numeric code to use for "Other" meta-program.      |
 
 ### Query String Parameters
 
-A couple query-string parameters are available to configure response output.
+Several query-string parameters are available for either endpoint to configure
+response output.
 
-An `enable_code_filter` parameter is available for the `/accounts` endpoint to
-optionally deduplicate codes, removing inactive codes and `CodesToOmit`.
-Defining any non-false value for this parameter will enable it.
+An `enable_other_code` parameter is available to optionally include an "Other"
+entry in the output with a value from the `OtherCode` parameter. Defining any
+non-false value for this parameter will enable it.
 
-A `disable_inactive_filter` parameter is available to modify the behavior of
-`enable_code_filter`. If this parameter is set to any non-false value, inactive
-codes are no longer removed. Codes are still deduplicated and `CodesToOmit` is
-respected. This parameter has no effect if `enable_code_filter` is not enabled.
+A `disable_no_program_code` parameter is available to optionally exclude a "No
+Program" entry from the output, otherwise it will be injected with a value from
+the `NoProgramCode` parameter. Defining any non-false value for this parameter
+will enable it.
 
-An `enable_other_code` parameter is available for either endpoint to optionally
-include an "Other" entry in the output with a value from the `OtherCode`
-parameter. Defining any non-false value for this parameter will enable it.
+A `disable_inactive_filter` parameter is available to modify how inactive codes
+are treated. If this parameter is set to any non-false value, inactive codes are
+no longer removed. Codes are still deduplicated and `CodesToOmit` is respected.
```

Comment:
what does `.. inactive codes are no longer removed` mean?  what is it no longer removed from?  do you mean from the API request?

Reply:
It's describing a change from the default behavior, I'll also add a description of that default behavior